src/CheckoutSdk/Sources/SourceData.cs

```diff
@@ -0,0 +1,46 @@
+using Newtonsoft.Json;
+
+namespace Checkout.Sources
+{
+    /// <summary>
+    /// Defines the source data of a <see cref="SourceRequest"/>.
+    /// </summary>
+    public class SourceData
```

Comment:
Source data is dynamic and can vary between sources. This should be treated the same as `AlternativePaymentSource`